content/influxdb/v2.0/reference/cli/influx/v1/auth/active.md

```diff
@@ -0,0 +1,31 @@
+---
+title: influx v1 auth active
+description: >
+  The `influx v1 auth active` command activates an authorization in InfluxDB.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth active
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth active` command activates an authorization in InfluxDB.
```

Comment:
Does this create an active authorization...it may read like this activates an existing authorization (which maybe is the case?)

Reply:
This is changing to `set-active` with https://github.com/influxdata/influxdb/pull/19935.